roles/dns_server/templates/192.168.1

```diff
@@ -0,0 +1,13 @@
+$TTL 60  ; 1 minute
+@		IN SOA	dns.{{ dns_domain }}. hostmaster.{{ dns_domain }}. (
+				2012113000 ; serial
```

Comment:
At the very least, we'll need serial to be a variable here...  how to track that variable, now that is trickier.  

My initial though is that we can naively use the date, but then we'd be bouncing named everytime once a day if ansible is being used to manage the environment.  alternatively, we can keep track of a counter, and increment the counter when we see that values have changed...  we can use "local facts" (https://docs.ansible.com/playbooks_variables.html#local-facts-facts-d)  to track this info on the dns server host.